benchmarking_sbi/README.md

```diff
@@ -21,51 +20,49 @@ The subfolder `results/` contains csv files with results for the manuscript *Ben
 | `supplement_hyperparameters_snpe.csv`         | Hyperparameters for (S)NPE                          |
 | `supplement_hyperparameters_snre.csv`         | Hyperparameters for (S)NRE                          |
 
-
 ## Raw results for each run
 
 Raw results for each single run from which these dataframes were compiled—including samples from all approximate posteriors—are also available. For space reasons, they are hosted in a separately using LFS:
 
-> https://github.com/mackelab/benchmarking_sbi_runs
-
+> <https://github.com/mackelab/benchmarking_sbi_runs>
 
 ## Reproducing results
 
 If you want to reproduce the experiments of the benchmark, or want to adapt our code to run your own experiments using the same analysis pipeline, start by cloning this repository:
 
 ```commandline
-$ git clone git@github.com:sbi-benchmark/results.git
```

Comment:
nit: For consistency with READMEs in other sbi-benchmark repos, I'd prefer keeping the $ signs indicating the prompt.